modules/cms/classes/ComponentPartial.php

```diff
@@ -107,6 +107,26 @@ public static function loadOverrideCached($theme, $component, $fileName)
             $partial = Partial::loadCached($theme, $component->alias . '/' . $fileName);
         }
 
+        if ($partial === null) {
+            $partial = Partial::loadCached($theme, strtolower($component->name) . '/' . $fileName);
```

Comment:
You are using `$component->name` directly, but isn't this attribute something like `Acme\Plugin\Components\Component`? I tried to deal with the same fallback [this way](https://github.com/octobercms/october/pull/4677/files#diff-be1899dc8e821d54db6ebb064918f0540328a1f3d9b02f0473259f873b23a938R111).

Reply:
I've just tested, `$component->name` just return the component name as expected:
```
[demoTodo demoTodoAlias]
```
returns `demoTodoAlias`

In most cases it should work but yeah, you are right: [according to the doc](https://octobercms.com/docs/cms/components#aliases), to avoid an eventual conflict, it could be a full class name including the namespace :+1:  :
```
[AuthorName\PluginName\Components\DemoTodo demoTodoAlias]
```
returns `AuthorName\PluginName\Components\DemoTodo`.

In those cases, your way seems to be the way, but I also see one case where your way will break: using the full path of a component means we are dealing with two plugins components which have the same name.

Knowing that it means that your way would try to load the partials of the two components from the same folder, it's right?
```
[AuthorOneName\MyAwesomePlugin\Components\DemoTodo awesomeDemoTodo]
[AuthorTwoName\PluginTwoName\Components\DemoTodo]
```
both of these statement will try to look into the `demotodo` folder, am I right?

What's your thoughts about the best of our two worlds?
```
strtolower(str_replace(['Components\\', '\\'], ['', '/'], $component->name))
```
would returns:
- `authorname/pluginname/demotodo` when used with `[AuthorName\PluginName\Components\DemoTodo demoTodoAlias]`
- `demotodo` when used with `[DemoTodo demoTodoAlias]`

So we can - *using the previous example where there was two DemoTodo components from two differents plugins* - load partials from:
- authoronename/myawesomeplugin/demotodo
- authortwoname/plugintwoname/demotodo

Do you agree with all my points?

Reply:
@RomainMazB I am still not sure if `$component->name` should be anything else than `AuthorName\PluginName\Components\DemoTodo`. How did you achieve that? If I want to find component's short name, which component got when registered (`PluginName::registerComponents()`), I have to call ComponentManager to ask him. And this step still missing here, so it looks like all my components will fall through theme partials overriding process (by component's registered short code) to their default templates (or to `themes/themename/partials/authoronename/pluginaname/demotodo` in your PR). Is good idea to ignore component's registered short code and send them automatically to `authoronename/pluginaname/demotodo` subfolder?

My previous PR was handy for me and reusing theme's partial design for many snippets of the same component, but it might not be handy for everyone :-)

Reply:
I've committed some new change, joined by [a new repository](https://github.com/RomainMazB/partials-override) so everyone can test easily the modifications.

I'll be happy to continue the discussion on the new commit ;)